data/blog/electron-3-0.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Electron 3.0.0
+author: codebytere
+date: '2018-09-20'
+---
+
+The Electron team is excited to annouce that the first stable release of Electron `v3.0` is now
+available from [electronjs.org](https://electronjs.org/) and via `npm install electron@latest`! It's jam-packed with upgrades, fixes, and new features, and we can't wait to see what you build with them. Below are details about this release, and we welcome your feedback as you explore.
+
+---
+
+## Release Process
+
+As we undertook development of `v3.0.0`, we sought to more empirically define criteria for a stable release by formalizing the feedback progress for progressive beta releases. `v3.0.0` would not have been possible without our [App Feedback Program](LINK-TBD) partners, who provided early testing and feedback during the beta cycle. Thanks to Atlassian, Atom, Microsoft Teams, Oculus, OpenFin, Slack, Symphony, VS Code, and other program members for their work. If you'd like to participate in future betas, please mail us at [info@electronjs.org](mailto:info@electronjs.org).
```

Comment:
LINK-TBD should be

https://electronjs.org/docs/tutorial/app-feedback-program 

(I think?)